.buildconfig.yml

```diff
@@ -96,6 +96,10 @@ projects:
     path: components/feature/top-sites
     description: 'Feature implementation for saving and removing top sites.'
     publish: true
+  feature-recents:
+    path: components/feature/recents
+    description: 'Feature implementation for saving and sorting recent apps.'
+    publish: true
```

Comment:
nit: This alone probably doesn't deserve its own module. How about using the name `feature-share` and having things like a customizable share sheet in here too in the future?

Reply:
Changed the naming to feature-share